### PR TITLE
WT-6246 Rename time pair to time point and pass correct stop ts to hs cell

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

File: src/include/timestamp.h [hunk 1]
```diff
@@ -16,14 +16,14 @@
 #define WT_TS_INT_STRING_SIZE (2 * 10 + 4 + 1)
```
Comment (agorrod) [inline src/include/timestamp.h:16]: Does this need to be updated to account for the durable time now?

File: src/reconcile/rec_col.c [hunk 2]
```diff
@@ -972,17 +963,10 @@ __wt_rec_col_var(
...
-                     * tombstone to write to disk and the deletion of the keys must be globally
-                     * visible.
+                     * The time window for deleted keys must be empty.
                      */
-                    WT_ASSERT(session,
-                      (!deleted && !last.deleted) ||
-                        (last.tw.durable_start_ts == tw.durable_start_ts &&
-                          last.tw.start_ts == WT_TS_NONE && last.tw.start_txn == WT_TXN_NONE &&
-                          last.tw.durable_stop_ts == tw.durable_stop_ts &&
-                          !WT_TIME_WINDOW_HAS_STOP(&last.tw)));
```
Comment (agorrod) [inline src/reconcile/rec_col.c:985]: Did you make this change deliberately? The old check seemed a bit odd to me, but it was comparing values between `last` and `tw` - which I'd presumed were deliberate.

File: src/history/hs.c [hunk 3]
```diff
@@ -364,7 +365,7 @@ __hs_insert_updates_verbose(WT_SESSION_IMPL *session, WT_BTREE *btree)
 static int
 __hs_insert_record_with_btree_int(WT_SESSION_IMPL *session, WT_CURSOR *cursor, WT_BTREE *btree,
   const WT_ITEM *key, const WT_UPDATE *upd, const uint8_t type, const WT_ITEM *hs_value,
-  WT_HS_TIME_PAIR stop_ts_pair)
```
Comment (agorrod) [inline src/history/hs.c:367]: You didn't add this here, but it's unusual to pass a structure as a parameter - we usuall pass as a pointer. Could you update this usage to do that? i.e: go from `WT_HS_TIME_POINT stop_time_point` to `WT_HS_TIME_POINT *stop_time_point` and similarly for below.

File: src/reconcile/rec_col.c (hunk 2 above)
Comment (quchenhao) [reply src/reconcile/rec_col.c:985]: I'm not sure why the code was wrote like that before but we already compare the two in line 970 so I think the change doesn't change the behavior.

File: src/include/timestamp.h (hunk 1 above)
Comment (quchenhao) [reply src/include/timestamp.h:16]: Based on the comment, it already take the durable time into account.

Comment (quchenhao) [reply src/include/timestamp.h:16]: Yes. We should change the size of WT_TP_STRING_SIZE

File: src/reconcile/rec_col.c (hunk 2 above)
Comment (agorrod) [reply src/reconcile/rec_col.c:985]: Got it - thanks!

**Pull request merged**